pkg/cmd/provisioning/gcp/create_service_accounts.go

```diff
@@ -285,9 +286,26 @@ func createServiceAccount(ctx context.Context, client gcp.Client, name string, c
 
 		// Add member <-> role bindings for the project
 		svcAcctBindingName := actuator.ServiceAccountBindingName(serviceAccount)
-		err = actuator.EnsurePolicyBindingsForProject(client, roles, svcAcctBindingName)
-		if err != nil {
-			return "", errors.Wrap(err, fmt.Sprintf("Failed to add predefined roles for IAM service account %s", serviceAccount.DisplayName))
+		// EnsurePolicyBindingsForProject can fail due to a replication delay after service account creation
+		// Try up to 24 times with a 10 second delay between each attempt, up to 4 minutes.
+		for i := 0; i < 12; i++ {
```

Comment:
should it be `i < 24` ?

Reply:
It actually should't be anything there... instead relying on either `err != nil` or `i >= 23` later to break out. [Fixed](https://github.com/openshift/cloud-credential-operator/compare/9878a519c2ca510c37d3d5d3afa7f59a2ebe42e3..70d1772ab948b3a25d1f6750af0769c370e74e4f).